tests/test_ISOSR.py

```diff
@@ -198,7 +198,7 @@ def test_attach_with_smb_credentials_domain(self, context,
         context.setup_error_codes()
         update = {'cifspassword': 'winter2019'}
         smbsr = self.create_smbisosr(atype='cifs', vers='1.0',
-                                     username='citrix\jsmith',
+                                     username=r'citrix\jsmith',
```

Comment:
this is not strictly a correct use of an `r` string, it should really just be either `citrix\\jsmith` or drop the domain piece entirely

Reply:
Why so ? Raw string literals are most commonly used for regexps, but there's no reason I know of to avoid them in other contexts.  Although in the python standard library doc there are only mentions in the regexp context, [the language reference](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals) has no such restriction.

Reply:
OK, fair enough. I was only ever introduced to these as being a regular expression declarator.

Reply:
`r` means raw, not regex.  Regexes tend to contain a lot of `\` which is why it is common for them to be written as raw strings, but they're just strings
```python
>>> type(r"")
<class 'str'>
```